pulse_actions/handlers/talos.py

```diff
@@ -0,0 +1,70 @@
+"""
+This module is for the following use case:
+
+ - Talos jobs:
+
+    * Trigger talos jobs twice if they are from PGO build.
+"""
+import logging
+
+from mozci import query_jobs
+from pulse_actions.utils.misc import filter_invalid_builders
+from mozci.errors import PushlogError
+from mozci.mozci import trigger_talos_jobs_for_build
+from mozci.platforms import get_buildername_metadata
+from mozci.sources import buildjson
+
+LOG = logging.getLogger()
+
+
+def on_event(data, message, dry_run):
+    """Trigger talos jobs twice in PGO builds."""
```

Comment:
Could you please say that we're listening for the completion of PGO builds on mozilla-inbound and fx-team in order that we trigger talos jobs twice? This is simply more verbose.